client/src/QAComponents/Question.jsx

```diff
@@ -56,11 +53,20 @@ const Question = (props) => {
 
   const addHelpfulQuestion = (e) => {
     e.preventDefault()
-    axios.put('/qa/helpfulquestion?question_id=' + e.target.parentNode.id, {
-      question_id: e.target.id
-    })
-      .then(data => setQHelpfulness(prev => prev++))
-      .catch(err => { console.error(err) })
+    if (!helpful) {
+      setHelpful(true)
+      axios.put('/qa/helpfulquestion?question_id=' + e.target.parentNode.parentNode.parentNode.parentNode.parentNode.id, {
+        question_id: e.target.id
+      })
+        .then(data => {
+          setQHelpfulness(prev => prev++)
+          return console.log('helpful', helpful)
```

Comment:
Console.logs have their place in bug testing, but eventually you may want to phase them out once your components start working (post tests, etc)

Reply:
Thanks! I got rid of them on initial load, but as I go through the code after deployment, I'll make sure they are all gone.